src/encoded/schemas/experiment_type.json

```diff
@@ -0,0 +1,151 @@
+        },
+        "controlled_term": {
+            "title": "Ontology Term",
+            "description": "Ontology term that matches this experiment type",
+            "type": "string",
+            "linkTo": "OntologyTerm"
+        },
+        "experiment_category": {
+            "title": "Experiment Category",
+            "description": "General category of the experiment type",
+            "type": "string",
+            "enum": ["Sequencing", "Microscopy"],
+            "lookup": 100
+        },
+        "assay_classification": {
+            "title": "Assay Classification",
+            "description": "Classification of assay type",
+            "type": "string",
+            "suggested_enum": [
+                "Linear DNA Enrichment",
+                "3C via Ligation",
+                "Ligation-free Multi-fragment 3C",
+                "RNA Quantification",
+                "RNA-DNA Proximity Capture",
+                "Fluorescence Localization"
+            ]
+        },
+        "assay_subclassification": {
+            "title": "Assay Sub-classification",
+            "description": "Sub-classification of assay based on what is measured",
+            "type": "string",
+            "suggested_enum": [
+                "Replication Timing",
+                "Proximity to Organelle",
+                "DNA Binding",
+                "Open Chromatin",
+                "DNA-DNA Pairwise Interactions",
+                "DNA-DNA Pairwise Interactions - Single Cell",
+                "DNA-DNA Multi-way Interactions",
+                "DNA-DNA Multi-way Interactions of Selected Loci",
+                "DNA-DNA Pairwise Interactions of Enriched Regions",
+                "DNA-DNA Pairwise Interactions of Selected Loci",
+                "RNA-DNA Multi-way Interactions",
+                "Fixed Sample DNA Localization",
+                "Fixed Sample RNA Localization",
+                "Time-Lapse"
+            ]
+        },
+        "other_tags": {
+            "title": "Other Tags",
+            "description": "Other tags that categorize or describe the experiment type",
+            "type": "array",
+            "items": {
+                "title": "Tag",
+                "description": "A tag for the item.",
+                "type":  "string"
+            }
+        },
+        "raw_file_types": {
+            "title": "Raw Files",
+            "description": "The type of raw files typically associated with the experiment type in the portal",
+            "type": "string",
+            "suggested_enum": ["Reads (fastq) provided by lab"]
+        },
+        "sop": {
+            "title": "SOP",
+            "description": "Approved SOP for the experiment type",
+            "type": "string",
+            "linkTo": "Protocol"
+        },
+        "references": {
+            "title": "Reference Publications",
+            "description": "Reference publication(s) for the experiment type",
+            "type": "string"
+        }
```

Comment:
We have a depreciated field 'references' that is link to , which we decided to replace with urls. We can do the same here , keeping the title same, but changing the property key (and maybe adding a format = url, ie
```
"url": {
            "title": "Reference Publications/Sources",
            "description": "Reference publication(s) for the experiment type",
            "type": "string",
            "lookup": 1000,
            "format": "uri"
        }```